react/src/views/Box/components/BoxDetails.tsx

```diff
@@ -221,7 +219,14 @@ function BoxDetails({
                 isChecked={boxData.state === BoxState.Scrap}
                 data-testid="box-scrap-btn"
                 isFocusable={false}
-                onChange={onScrap}
+                onChange={() =>
+                  onStateChange(
+                    // toggle the box state
```

Comment:
Can you put a more precise comment, like "If the current box state 'Scrap' is toggled, set the defaultBoxState of the box location"? If my understanding is correct